GSoC25_H/llm_IE/llm_interface.py

```
@@ -18,97 +21,56 @@ class ExtractionResult:
     error: Optional[str] = None
 
 class OllamaInterface:
-    """Unified interface for interacting with Ollama models"""
+    """
+    Unified interface for interacting with Ollama models.
+    Refactored to use the shared src.llm_core.LLMService instead of raw requests.
+    """
 
     def __init__(self, model_config: ModelConfig, base_url: str = "http://localhost:11434"):
```

Ruff (0.14.13)

Unused method argument: `base_url`

(ARG002)